scripts/rosa/delete-rosa-clusters.sh

```diff
@@ -102,9 +102,42 @@ delete_old_clusters() {
     done
 }
 
+delete_target_groups() {
+    local today
+    local all_target_groups
+    local todays_target_groups
+    local todays_target_group_array
+    local tg_arn
+
+    # Delete last 24hours
+    today=$(date -u -d '24 hours ago' +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Is it not possible to delete specific target groups tied to load balancers that are about to be deleted, instead of deleting all target groups older than 1 day?

I know that we currently have one region fully dedicated to test clusters, so this approach wouldn't be a problem now, but wondering what if this changes in a future. Or what if somebody from other team would use this script.

Reply:
@psturc good point. Can you check the new code?